Project.toml

```diff
@@ -12,21 +12,6 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 ChainRulesCore = "1.11.5"
-ChainRulesTestUtils = "1"
 Compat = "3.35"
```

Comment:
Likely this can drop Compat.jl for now. Although surely it'll be needed for something again soon, so less churn may be better?

Reply:
yeah lets drop it.